src/Mollie/WC/Payment/Object.php

```diff
@@ -642,5 +642,21 @@ public function deleteSubscriptionOrderFromPendingPaymentQueue( $order ) {
 
 		}
 	}
+    /**
+     * @param WC_Order $order
+     *
+     * @return bool
+     */
+    protected function isFinalOrderStatus(WC_Order $order)
+    {
+        $data_helper = getDataHelper();
+        $orderStatus = $data_helper->getOrderStatus($order);
+        $isFinalOrderStatus = \in_array(
+            $orderStatus,
+            ['completed', 'refunded', 'canceled'],
```

Comment:
I would use a constant for this list, that would be useful in case we need to reuse the same list somewhere else.